react-ui/src/pages/contest/Contest.jsx

```diff
@@ -314,450 +121,136 @@ function Contest() {
     );
   }, [selectedCategories]);
 
-  const handleCategoryChange = (event) => {
-    setSelectedCategories(event.target.value.sort());
-  };
-
-  const resetSelectedCategories = () => {
-    setSelectedCategories([]);
-  };
-
-  const viewDescription = (entryId) => {
-    setDescriptionEntryId(entryId);
-    setSettingsOpen(true);
-  };
+  const handleVotingExpired = useCallback(() => {
+    updateCache(null);
+    setVotingExpired(true);
+  }, []);
 
-  const [{ density = 'default' }, updateSettings] = useSettingsState();
+  const handleReload = useCallback(() => {
+    scrollInstantlyTo(0);
+    window.location.reload();
+  }, []);
 
-  const handleDensityChange = (event) => {
-    updateSettings('density', event.target.value);
-  };
+  const toggleDrawerOpen = useCallback((isOpen) => {
+    setExperimentId(null);
+    setIsDrawerOpen(isOpen);
+  }, []);
 
-  const toggleSettingsOpen = () => {
-    setSettingsOpen(!isSettingsOpen);
-  };
+  const closeDrawer = useCallback(() => {
+    toggleDrawerOpen(false);
+  }, []);
 
-  const backLink = (state || {}).back || '/contests';
+  const setExperimentDrawer = useCallback((entryId) => {
```

Comment:
Also, this will eventually be changed from an experiment to permanent functionality. We ideally wouldn't have to go and rename everything just because it's currently being used as an experiment.

Reply:
See above